presto-product-tests/conf/presto/etc/config.properties

```diff
@@ -1,47 +0,0 @@
-#
```

Comment:
Please update product test README. Instead of mentioning how one can run Presto from IDE, we should mention how they can attach debugger to Presto from docker.